views/place-form.hbs

```diff
@@ -1,211 +1,226 @@
-
-  <fieldset name="en" class="">
-    <legend>{{localText.english}}</legend>
-    <label>{{localText.name}}
-      {{#if place.en.name}}
-        <input type="text" name="name_en" required value="{{place.en.name}}">
-      {{else}}
-        <input type="text" name="name_en" required>
-      {{/if}}
-    </label>
-
-    <label>{{localText.description}}
-      {{#if place.en.description}}
-        <input type="text" name="description_en" required value="{{place.en.description}}">
-      {{else}}
-        <input type="text" name="description_en" required>
-      {{/if}}
-    </label>
-
-    <label>{{localText.placeAddress}}
-      {{#if place.en.address}}
-        <input type="text" name="address_en" required value="{{place.en.address}}">
-      {{else}}
-        <input type="text" name="address_en" required>
-      {{/if}}
-    </label>
-
-    <label>{{localText.placeHours}}
-      {{#if place.en.openingHours}}
-        <input type="text" name="openingHours_en" required value="{{place.en.openingHours}}">
-      {{else}}
-        <input type="text" name="openingHours_en" required>
-      {{/if}}
-    </label>
-  </fieldset>
-
-  <fieldset name="ar" class="">
-    <legend>{{localText.arabic}}</legend>
-    <label>{{localText.name}}
-      {{#if place.ar.name}}
-        <input type="text" name="name_ar" required value="{{place.ar.name}}">
-      {{else}}
-        <input type="text" name="name_ar" required>
-      {{/if}}
-    </label>
-
-    <label>{{localText.description}}
-      {{#if place.ar.description}}
-        <input type="text" name="description_ar" required value="{{place.ar.description}}">
-      {{else}}
-        <input type="text" name="description_ar" required>
-      {{/if}}
-    </label>
-
-    <label>{{localText.placeAddress}}
-      {{#if place.ar.address}}
-        <input type="text" name="address_ar" required value="{{place.ar.address}}">
-      {{else}}
-        <input type="text" name="address_ar" required>
-      {{/if}}
-    </label>
-
-    <label>{{localText.placeHours}}
-      {{#if place.ar.openingHours}}
-        <input type="text" name="openingHours_ar" required value="{{place.ar.openingHours}}">
-      {{else}}
-        <input type="text" name="openingHours_ar" required>
-      {{/if}}
-    </label>
-  </fieldset>
-
-  <label>{{localText.formOwnerId}}
-    {{#if place.ownerId}}
-      <input type="text" name="ownerId" value="{{place.ownerId}}">
-    {{else}}
-      <input type="text" name="ownerId">
-    {{/if}}
-  </label>
-
-  <label>{{localText.formImageUrl}}
-    {{#if place.imageUrl}}
-      <input type="text" name="imageUrl" value="{{place.imageUrl}}">
-    {{else}}
-      <input type="text" name="imageUrl">
-    {{/if}}
-  </label>
-
-  <label>{{localText.placeWebsite}}
-    {{#if place.website}}
-      <input type="text" name="website" value="{{place.website}}">
-    {{else}}
-      <input type="text" name="website">
-    {{/if}}
-  </label>
-
-  <label>{{localText.placePhone}}
```

Comment:
You missed the phone number field (btw I would make it a `type='text'` I think it makes for a better UX